config/settings.yml

```diff
@@ -1386,6 +1386,7 @@ vanotify:
       api_key: fake_secret
       template_id:
         form1010_ez_failure_email: fake_template_id
+        form1010_ezr_failure_email: fake_template_id
```

Comment:
Just wanted to call out that if you haven't created it yet you'll need to update the `vsp-infra-application-manifests` repo similar to [this PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2923).